tests/suites/casper/visible.js

```diff
@@ -2,11 +2,11 @@
 /*jshint strict:false*/
 casper.test.begin('visibility tests', 5, function(test) {
     casper.start('tests/site/visible.html', function() {
-        test.assert(!this.visible('#img1'), 'Casper.visible() can detect if an element is invisible');
+        test.assert(this.visible('#img1'), 'Casper.visible() can detect if an element is invisible');
         test.assert(this.visible('#img2'), 'Casper.visible() can detect if an element is visible');
         test.assert(!this.visible('#img3'), 'Casper.visible() can detect if an element is invisible');
         test.assert(this.visible('img'), 'Casper.visible() can detect if an element is visible');
-        this.waitWhileVisible('#img1', function() {
+        this.waitUntilVisible('#img1', function() {
             test.pass('Casper.waitWhileVisible() can wait while an element is visible');
```

Comment:
nope.
[waitUntilVisible](http://casperjs.readthedocs.org/en/latest/modules/casper.html?highlight=waituntilvisible#waituntilvisible) and [waitWhileVisible](http://casperjs.readthedocs.org/en/latest/modules/casper.html?highlight=waituntilvisible#waitwhilevisible) don't have the same behavior.